processor/src/chiplets/hasher/mod.rs

```diff
@@ -189,10 +211,13 @@ impl Hasher {
     /// The returned tuple also contains the row address of the execution trace at which the hash
     /// computation started and the lookups required to verify the computation so that the correct
     /// requests can be sent by the caller to the Chiplets Bus.
+    /// TODO: Refactor to require fewer is_memoized checks. This can be done if the intermediary
+    /// states don't need to be passed to the lookup.
```

Comment:
Let's create an issue for this.

Reply:
This is part of the hasher lookup refactor issue that @tohrnii is also working on.

Reply:
As @grjte said, it's added to the hasher lookup refactor issue.